### PR TITLE
fixed link names and adding privacy name

### PR DESCRIPTION
**Pull request by MartinSchuhmacher**
# Description

Fixed the link names for privacy and termsOfUse on homepage of Niedersachsen, Thueringen and Open. Additionally added name for privacy link in collection.

## Links to Tickets or other pull requests

https://ticketsystem.hpi-schul-cloud.org/browse/SC-6931
## Changes

- correct links in footers
- added privacy name for links

## Data Security <sub><sup>details [on Confluence](https://docs.schul-cloud.org/x/2S3GBg)</sup></sub>

No data security topics
## Screenshots of UI changes

image
image
image

## Approval for review
- [x] All points were discussed with the ticket creator, support-team or product owner. The code upholds all quality guidelines from the PR-template.

> Notice: Please remove the WIP label if the PR is ready to review, otherwise nobody will review it.

### Link to Definition of Done
More and detailed information on the *definition of done* can be found [on Confluence](https://docs.schul-cloud.org/pages/viewpage.action?pageId=92831762)

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -7,6 +7,12 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 Allowed Types of change: `Added`, `Changed`, `Deprecated`, `Removed`, `Fixed`, `Security`
 
+## Unreleased
+
+## Fixed
+- SC-6931 fixed link names in Niedersachsen, Open and Thueringen to privacy and termsOfUse on homepage
+
+=======
```
Comment (ABialoskorski) [inline CHANGELOG.md:15]: This ========= is redundant, right?

Comment (MartinSchuhmacher) [reply CHANGELOG.md:15]: I guess so yes, but was not sure if this is convention or not, so I kept it in. Shall I delete it?

File: locales/de.json [hunk 2]
```diff
@@ -2348,6 +2348,7 @@
 			"firstLoginExistingUser": "Willkommen - Erster Login für bestehende Nutzer",
 			"loginFailedHint": "Deine Anmeldung hat leider nicht geklappt, bitte versuche folgende Schritte zur Problemlösung",
 			"loginForRegisteredUsers": "Login für registrierte Nutzer:innen",
+			"onlyPrivacy": "Datenschutzerklärung",
```
Comment (ABialoskorski) [inline locales/de.json:2351]: You can use already existing key: global.text.dataProtection

File: locales/en.json [hunk 3]
```diff
@@ -2348,6 +2348,7 @@
 			"firstLoginExistingUser": "Welcome - First login for existing users",
 			"loginFailedHint": "Unfortunately, your Login did not work, please try the following steps to solve the problem",
 			"loginForRegisteredUsers": "Login for registered users",
+			"onlyPrivacy": "Privacy Policy",
```
Comment (ABialoskorski) [inline locales/en.json:2351]: You can use already existing key: global.text.dataProtection

**Review by ABialoskorski — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (ABialoskorski) [reply CHANGELOG.md:15]: please delete it :)

**Review by ABialoskorski — APPROVED** [review]

**Review by mwolowyk-cap — APPROVED** [review]

**Review by sarahschaefer — APPROVED** [review]

**Pull request merged**